### ISSUE TITLE
mibew-mattermost plugin

### ISSUE THREAD

**Issue by artemisapuci** [Mibew/mattermost-plugin#1]
Hello I want to create an integration between mibew and mattermost, I have already fulfilled all of the steps in the installation document ,installed mibew messenger, put the files in the mibew/plugins and did the plugin configuration in the <Mibew root>/configs/config.yml file but when I go to <Mibew Base URL>/operator/plugin to enable the plugin, no plugin is showed , do you have any idea on what the problem may be ? Thanks

**Comment by tzeumer** [comment]
Hello. 

just a wild guess, but is it really in the right folder? The first time I used a plugin I got confused by the "double Mibew" in the path plugins/Mibew/Mibew/Plugin.

![image](https://user-images.githubusercontent.com/119825/141158770-3106cf02-15c3-4ad0-ad08-877de3d05171.png)

**Comment by faf** [comment]
Yes, most likely it's the paths. That's a very common trouble, despite the fact that this feature is described in the docs: https://docs.mibew.org/plugins/installation.html :)

Another possible reason could be in invalid permissions on dirs and/or files. For example if web server could not read (or execute) the dir of a plugin, Mibew Messenger would be unable to "see" it.

**Comment by artemisapuci** [comment]
Thanks
